insights/combiners/tomcat_virtual_dir_context.py

```diff
@@ -0,0 +1,39 @@
+"""
+TomcatVirtualDirContextCombined - check VirtualDirContext option used in Tomcat
+===============================================================================
+
+This combiner provides information about whether a VirtualDirContext is used in config files in
+both default locations or location derived from running Tomcat process.
+
+Examples::
+
+    >>> shared[TomcatVirtualDirContextCombined].data
+    {'/usr/share/tomcat/conf/server.xml':
+     ['    <Resources className="org.apache.naming.resources.VirtualDirContext"'],
+     }
+"""
+from ..core.plugins import combiner
+from ..parsers.tomcat_virtual_dir_context import TomcatVirtualDirContextFallback, \
+    TomcatVirtualDirContextTargeted
+
+
+@combiner(requires=[[TomcatVirtualDirContextFallback, TomcatVirtualDirContextTargeted]])
+class TomcatVirtualDirContextCombined(object):
+    """
+    Combiner for VirtualDirContext usage in Tomcat config files.
+    """
+    def __init__(self, local, shared):
```

Comment:
Intead of `local, shared`, the arguments to `__init__` are the instance objects for the classes you specify in the `@combiner` decorator.  So for instance:
```
def __init__(self, tomcat_vdc_fallback, tomcat_vdc_targeted):
```